### ISSUE TITLE
webp not working

### ISSUE THREAD

**Issue by bobhuf** [timber/timber#1849]
### Expected behavior
webp generation

### Actual behavior
jpg is generated in stead...

### Steps to reproduce behavior
Used the basic example you [posted on twitter](https://twitter.com/TimberWP/status/1042066512813744129):
```
{% set image = Image(50) %}
<picture>
  <source srcset="{{ image.src | towebp }}" type="image/webp">
  <img src="{{ image.src }}" alt="{{ image.alt }}">
</picture>
```

### What version of WordPress, PHP and Timber are you using?
WordPress 4.9.8, PHP 7.1.12, Timber 1.8.1

**Comment by palmiak** [comment]
Hi :)

First of all:
``` 
var_dump( gd_info() );
```
Let's be sure your server has webp support.

**Comment by pascalknecht** [comment]
Yes your server probably does not support the image extension needed for the conversion. You may want to check with your hoster.

**Comment by jarednova** [comment]
Thanks for the report @bobhuf! It seems that this is a server issue with your `gd` or `imagick`. That said, perhaps we can add something to the error logs alerting the dev to WebP not being supported

**Comment by bobhuf** [comment]
You are all right, my local setup (MAMP) has GD enabled but without webp support. Everything works as expected on our dev servers. Maybe add some more info about requirements to the docs as well?

**Comment by jarednova** [comment]
I just submitted #1854 to resolve the documentation gap

**Comment by jarednova** [comment]
Closed by #1854
